scripts/scenario_03_waves.lua

```diff
@@ -122,7 +122,7 @@ end
 
 function spawnWave()
     waveNumber = waveNumber + 1
-    getPlayerShip(-1):addToShipLog("Wave " .. waveNumber, "red")
+    getPlayerShip(-1):addToShipLog(_("Wave ") .. waveNumber, "red")
```

Comment:
The right way to add these markers is:
`string.format(_("Wave %d"), waveNumber)` so translators can decide of the number needs to come first or not.

Reply:
Like that?

Reply:
Done